mkt/search/views.py

```diff
@@ -101,13 +104,23 @@ def get_serializer_context(self):
         }
         return context
 
+    def _get_colombia_filter(self):
+        if self.allow_colombia and self.request.REGION == mkt.regions.COL:
+            return None
+        co_filter = es_filter.Term(tags=COLOMBIA_WEBSITE)
+        return es_filter.F(es_filter.Bool(must_not=[co_filter]),)
+
     def get_queryset(self):
         excluded_fields = list(set(WebappIndexer.hidden_fields +
                                    WebsiteIndexer.hidden_fields))
-        return (Search(using=BaseIndexer.get_es(),
-                       index=self._get_indices(),
-                       doc_type=self._get_doc_types())
-                .extra(_source={'exclude': excluded_fields}))
+        co_filters = self._get_colombia_filter()
+        qs = (Search(using=BaseIndexer.get_es(),
+                     index=self._get_indices(),
+                     doc_type=self._get_doc_types())
+              .extra(_source={'exclude': excluded_fields}))
+        if co_filters:
+            return qs.filter(co_filters)
```

Comment:
It appears to chain, but examples are scarce:

http://elasticsearch-dsl.readthedocs.org/en/latest/search_dsl.html#filters